docs/CHANGELOG.md

```diff
@@ -43,14 +41,24 @@ The following tip changes can be tested by building VictoriaMetrics components f
   `--search.maxGraphiteTagKeys` for limiting the number of tag keys returned from Graphite `/tags`, `/tags/autoComplete/*`, `/tags/findSeries` API. 
   `--search.maxGraphiteTagValues` for limiting the number of tag values returned Graphite `/tags/<tag_name>` API. 
   Remove redundant limit from [Prometheus api/v1/series](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#prometheus-querying-api-usage). See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4339).
+* BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): fix application routing issues and problems with manual URL changes. See [this pull request](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/4408).
```

Comment:
Isn't this bugfix included to 1.91.3? 
I see this commit tagged here https://github.com/VictoriaMetrics/VictoriaMetrics/commit/9843ec0e1d471dae7074d0b4b52a4af29651d245

Reply:
oh, that's different commit